.idea/compiler.xml

```diff
@@ -13,4 +13,9 @@
       <module name="spring-data-influxdb" target="1.8" />
     </bytecodeTargetLevel>
   </component>
+  <component name="JavacSettings">
```

Comment:
Any rationale why to add this?

Reply:
this is automatically appended by IDEA, if i remove it manual, it will be appended back again, maybe it is related with the version of idea,so it would be better to keep it.